pkg/common/jindo.go

```diff
@@ -20,15 +20,15 @@ package common
 const (
 	JindoRuntime = "jindo"
 
-	JindoChartName = "jindofs"
+	JindoChartName = "jindofsx"
 
 	JindoMountType = "fuse.jindofs-fuse"
 
 	JINDO_SMARTDATA_IMAGE_ENV = "JINDO_SMARTDATA_IMAGE_ENV"
 
 	JINDO_FUSE_IMAGE_ENV = "JINDO_FUSE_IMAGE_ENV"
 
-	DEFAULT_JINDO_RUNTIME_IMAGE = "registry.cn-shanghai.aliyuncs.com/jindofs/smartdata:3.8.0"
+	DEFAULT_JINDO_RUNTIME_IMAGE = "registry.cn-shanghai.aliyuncs.com/jindofs/smartdata:4.3.0"
```

Comment:
DEFAULT_JINDO_RUNTIME_IMAGE and DEFAULT_JINDOFSX_RUNTIME_IMAGE

Reply:
done